README.md

```diff
@@ -114,22 +114,21 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://deploy-preview-31--rentbikes.netlify.app)
+- [Live Demo Link]
```

Comment:
- [ ] In the "Remove a bike" section, the buttons have a spelling error:

![image](https://user-images.githubusercontent.com/57421823/230703818-cfa0ba62-5164-41d0-aa84-2b94e3171f60.png)

The buttons should read "Delete", instead of "Detele".   

Kindly fix this typo so your app looks more professional for English speakers. 💪